autoPyTorch/pipeline/components/setup/traditional_ml/traditional_learner/learners.py

```diff
@@ -45,8 +45,10 @@ def _prepare_model(self,
                        X_train: np.ndarray,
                        y_train: np.ndarray
                        ) -> None:
-        early_stopping = 150 if X_train.shape[0] > 10000 else max(round(150 * 10000 / X_train.shape[0]), 10)
-        self.config["early_stopping_rounds"] = early_stopping
+
+        if self.has_val_set:
+            early_stopping = 150 if X_train.shape[0] > 10000 else max(round(150 * 10000 / X_train.shape[0]), 10)
```

Comment:
We don't early stopping if `self.has_val_set` is set as False?

Reply:
yeah, we can't as we rely on external libraries to implement this for us